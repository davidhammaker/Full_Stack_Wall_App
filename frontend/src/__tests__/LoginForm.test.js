import React from 'react';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import renderer from 'react-test-renderer';
import LoginForm from '../components/loginform';


configure({ adapter: new Adapter() });


it('displays form correctly', () => {
  const loginForm = renderer.create(
    <LoginForm />
  );
  let tree = loginForm.toJSON();
  expect(tree).toMatchSnapshot();
});
