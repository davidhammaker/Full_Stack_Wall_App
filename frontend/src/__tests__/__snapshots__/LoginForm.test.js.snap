// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`displays form correctly 1`] = `
<form
  className="p-4"
  onSubmit={[Function]}
>
  <small />
  <div
    className="form-group"
  >
    <label>
      Username:
    </label>
    <input
      className="form-control"
      onChange={[Function]}
      type="text"
    />
  </div>
  <div
    className="form-group"
  >
    <label>
      Password:
    </label>
    <input
      className="form-control"
      onChange={[Function]}
      type="password"
    />
  </div>
  <div
    className="d-flex justify-content-end"
  >
    <input
      className="btn btn-info mt-2"
      type="submit"
      value="Submit"
    />
  </div>
</form>
`;
